extensions/wikia/AdEngine/js/AdSlotsOasis.js

```diff
@@ -1,24 +1,22 @@
-/*global define*/
-define('ext.wikia.adEngine.adDecoratorTopInContent', [
+/*global require*/
+require([
 	'wikia.log',
 	'wikia.document',
 	'ext.wikia.adEngine.adContext',
-	'ext.wikia.adEngine.adPlacementChecker'
-], function (log, document, adContext, adPlacementChecker) {
+	'ext.wikia.adEngine.adPlacementChecker',
+	'ext.wikia.adEngine.adSlotsInContent'
+], function (log, document, adContext, adPlacementChecker, adSlotsInContent) {
 	'use strict';
 
-	var logGroup = 'ext.wikia.adEngine.adDecoratorTopInContent',
-		slotName = 'TOP_INCONTENT_BOXAD';
-
 	function doesNotBreakContent() {
```

Comment:
I believe this function is not needed anymore, try launching the slots immediately, just like for Venus.